code/modules/clothing/suits/armor.dm

```diff
@@ -117,7 +117,6 @@
 	desc = "A suit of armor with heavy padding to protect against melee attacks. Looks like it might impair movement."
 	icon_state = "riot"
 	item_state = "swat_suit"
-	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS

```

Comment:
Is removing UPPER_TORSO|LOWER_TORSO intentional?

Reply:
Yes.